playbooks/ci-ceph_swift/tasks/pre-deploy.yml

```diff
@@ -35,6 +35,32 @@
     - name: umount ephemeral disk on ceph_osds
       shell: umount /dev/{{ item }}
       with_items: "{{ ceph.disks }}"
+      ignore_errors: true
+
+- name: umount ephemeral disk on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: umount ephemeral disk on swiftnode
+      shell: umount /dev/{{ item.disk }}
+      with_items: "{{ swift.disks }}"
+      ignore_errors: true
+
+- name: create new partition table for ephemeral disk on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: create new msdos partition table for ephemeral disk
+      shell: parted /dev/{{ item.disk }} --script mklabel msdos
+      with_items: "{{ swift.disks }}"
+
+- name: deploy fake lshw file on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: rename the original /usr/bin/lshw
+      command: mv /usr/bin/lshw /usr/bin/lshw.original
+    - name: deploy fake lshw file /usr/bin/lshw
+      template: src=../templates/usr/bin/lshw
+                dest=/usr/bin/lshw
+                owner=root group=root mode=0755
```

Comment:
I assume that update-alternatives should work (maybe I had some misunderstanding about how it works in the very beginning), but anyway, the benefit of my above approach is that it can minimize the impact of replacing the original lshw script by wrapping the output of original lshw in my fake one if needed (i.e. when it is not running `lshw -C disk`).